doc/nrf/known_issues.rst

```diff
@@ -220,6 +220,20 @@ Assert risk after performing a DLE procedure
 Other issues
 ------------
 
+.. rst-class:: v1-4-0 v1-3-2 v1-3-1 v1-3-0
+
+NCSDK-6844: Security procedure failure can terminate GATT client request
```

Comment:
Is it the case that the GATT client request comes from the same side that is initiating the security procedure?

Reply:
It does not have to be, the trigger is the encrypt change callback to ATT with an error code.
This originates from bt_conn_security_changed in the code. For example the HCI encrypt change event.

In this case it was triggered by SMP so that the security_changed callback could be given to the application since it initiated security.

```
	if (atomic_test_bit(smp->flags, SMP_FLAG_SEC_REQ)) {
		/* Notify higher level that security failed if security was
		 * initiated by slave.
		 */
		bt_conn_security_changed(conn, BT_HCI_ERR_PIN_OR_KEY_MISSING,
					 BT_SECURITY_ERR_PIN_OR_KEY_MISSING);
	}
```